modules/core/help.py

```
@@ -0,0 +1,295 @@
+        if Config('help_url', cfg_type=str):
+            help_msg.append(
+                msg.locale.t(
+                    "core.message.help.document",
+                    url=Config('help_url', cfg_type=str)))
+        await msg.finish(help_msg)
+
+
+env = Environment(loader=FileSystemLoader('assets/templates'))
+
+
+async def help_generator(msg: Bot.MessageSession, module_list: Dict[str, Module], target_enabled_list: Optional[List[str]] = [], show_base_modules: bool = True, show_disabled_modules: bool = False, use_local=True):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)